test/e2e/tests/metrics/delete-metametrics-data.spec.ts

```diff
@@ -230,7 +197,6 @@ describe('Delete MetaMetrics Data @no-mmi', function (this: Suite) {
         await driver.clickElement(selectors.accountOptionsMenuButton);
         await driver.clickElement(selectors.globalMenuSettingsButton);
         await driver.clickElement(selectors.securityAndPrivacySettings);
-        await driver.findElement(selectors.deletMetaMetricsSettings);
```

Comment:
This button wrapper section will always be present in the DOM, remove it will have less assertion to operate